app/Models/Tag.php

```diff
@@ -3,8 +3,11 @@
 namespace Rogue\Models;
 
 use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\SoftDeletes;
 
 class Tag extends Model
 {
-    //
+    use SoftDeletes;
```

Comment:
@DFurnes For some reason I thought we used that in the old system, but turns out we didn't! I don't think there's a use for it on the tags themselves, I'm removing it.